### ISSUE TITLE
[Accessibility]: Video tags without titles are undetectable

### ISSUE THREAD

**Issue by st0nebraker** [sourcegraph/about#5535]
### Audit type

Screen reader navigation

### User journey audit issue

#5500

### Problem description

The videos on the home page don't have a title, so they are skipped over entirely by the screen reader. Other video tags (`<video />`) may be missing these as well.

### Expected behavior

All rich content should be accessible, therefore video tags should have a title property to ensure screen readers pick up on them. Because the screen reader says "{Example title} group", we should include the content type "video" as a suffix to the video's title, so it can read "{Example title video} group" and make more sense.

### Assigning labels

- [ ] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [ ] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [ ] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.
